src/Monitor/Monitor/OutputClasses/PSPrivateEndpointConnection.cs

```diff
@@ -0,0 +1,97 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Rest;
+using Microsoft.Rest.Serialization;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Commands.Insights.OutputClasses
+{
+    /// <summary>
+    /// Initializes a new instance of the PrivateEndpointConnection class.
+    /// </summary>
+    /// 
+    [JsonTransformation]
+    public class PSPrivateEndpointConnection
```

Comment:
Is it required? because PrivateEndpointConnection is defined in network module

Reply:
yes, it need to be wrapped in private link scope